### PR TITLE
generalize bitcoin -> deposit addr options

### PR DESCRIPTION
**Pull request by dcposch**

### CODE REVIEW

File: packages/connectkit/src/components/Common/CopyToClipboard/CopyToClipboardIcon.tsx [hunk 1]
```diff
@@ -67,15 +67,17 @@ const IconContainer = styled(motion.div)<{ $clipboard?: boolean }>`
 const CopyToClipboardIcon = ({
   copied,
   small,
+  dark,
 }: {
   copied?: boolean;
   small?: boolean;
+  dark?: boolean;
```
Comment (dcposch) [inline packages/connectkit/src/components/Common/CopyToClipboard/CopyToClipboardIcon.tsx:74]: = dark mode?

File: packages/connectkit/src/components/Pages/WaitingDepositAddress/index.tsx [hunk 2]
```diff
@@ -39,32 +42,30 @@ const WaitingBitcoin: React.FC = () => {
...
     triggerResize();
-  }, [option]);
+  }, [details]);
 
   return (
     <PageContent>
       {failed ? (
         <ModalContent style={{ marginLeft: 24, marginRight: 24 }}>
-          <ModalH1>Bitcoin unavailable</ModalH1>
+          <ModalH1>{selectedDepositAddressOption} unavailable</ModalH1>
```
Comment (dcposch) [inline packages/connectkit/src/components/Pages/WaitingDepositAddress/index.tsx]: Further down you have `selectedDepositAddressOption.id` -> is this an object / can you use it as string here?

Maybe make a `chainName` var that is "Bitcoin" / "Tron" / etc

**Review by dcposch — COMMENTED** [review]
Nice cleanup, LGTM

File: packages/connectkit/src/components/Pages/WaitingDepositAddress/index.tsx (hunk 2 above)
Comment (nalinbhardwaj) [reply packages/connectkit/src/components/Pages/WaitingDepositAddress/index.tsx]: good catch, it was supposed to be `selectedDepositAddressOption.id` indeed, surprised typescript doesn't complain.

**Pull request merged**